Makefile

```diff
@@ -155,3 +156,19 @@ console:
 destroy: deploy-init
 	cd terraform && . workspace_variables/$(DEPLOY_ENV).sh \
 		&& terraform destroy -var-file=workspace_variables/$(DEPLOY_ENV).tfvars $(AUTO_APPROVE)
+
+enable-maintenance: ## make qa enable-maintenance / make prod enable-maintenance CONFIRM_PRODUCTION=y
+	$(if $(HOSTNAME), $(eval REAL_HOSTNAME=${HOSTNAME}), $(eval REAL_HOSTNAME=${APP_ENV}))
+	cf target -s ${SPACE}
```

Comment:
The space variable is lowercase in this makefile so this is failing.
Might as well set is uppercase for consistency? See line 104 etc